src/ProductConstructionService/ProductConstructionService.DependencyFlow/WorkItemProcessors/PullRequestCheckProcessor.cs

```diff
@@ -14,8 +14,10 @@ public override async Task<bool> ProcessWorkItemAsync(
         PullRequestCheck workItem,
         CancellationToken cancellationToken)
     {
-        var updater = _updaterFactory.CreatePullRequestUpdater(PullRequestUpdaterId.Parse(workItem.UpdaterId));
-        return await updater.CheckPullRequestAsync(workItem);
+        await Task.Delay(1);
```

Comment:
Revert this

Reply:
oops, forgot to change that when I validated that retries work, they do btw, and the dequeue count looks good, at least when I run the service locally